lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+  cardinality: 0..1
+  type: string
+  description: The credit card authorization number
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-authorization",
+      "valueString": "123456"
+    }
+  note: Optional field on payment transactions when method is Credit Card.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-authorization
+
+- name: financial-transaction-tendered-amount
+  required: 'C'
+  cardinality: 0..1
+  type: Money
+  description: Tendered amount of the cash payment.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount",
+      "valueMoney": {
+          "value": "123000",
+          "currency": {
+              "coding": [
+                  {
+                      "system": "urn:iso:std:iso:4217",
+                      "code": "USD",
+                      "display": "United States Dollar",
+                      "userSelected": true
+                  }
+              ],
+              "text": "United States Dollar"
+          }
+      }
+    }
+  note: Should be greater than or equal to the value of the financial-transaction-amount. Required on payment transactions when method is Cash.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount
+
+- name: financial-transaction-amount
+  required: 'Yes'
+  cardinality: 0..1
+  type: Money
+  description: The total amount of the FinancialTransaction resource.
+  action: create
+  example: |
+    {
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount",
+        "valueMoney": {
+            "value": "123000",
+            "currency": {
+                "coding": [
+                    {
+                        "system": "urn:iso:std:iso:4217",
+                        "code": "USD",
+                        "display": "United States Dollar",
+                        "userSelected": true
+                    }
+                ],
+                "text": "United States Dollar"
+            }
+        }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount
+
+- name: financial-transaction-allocation
+  required: 'Yes'
+  cardinality: 1..*
+  type: BackboneElement
+  description: How the FinancialTransaction should be applied.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation",
+      "extension": [
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-target",
+              "valueReference": {
+                  "reference": "Encounter/31363178"
+              }
+          },
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-amount",
```

Comment:
amount

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/61f2b0d5c518b3f2d50b893fdc1e6082f1093e4c#diff-26870e1b94fc9955e130f3f940dbe2e7R232

Reply:
@ReaganYuan This still needs to be done